configure.ac

```diff
@@ -120,6 +120,10 @@ GEANY_CHECK_THE_FORCE dnl hehe
 # i18n
 GEANY_I18N
 
+AM_COND_IF([MINGW],
+  [docdir='${prefix}/doc'],
+  [docdir='${docdir}'])
```

Comment:
As the Travis failure shows, this is a recursive variable from Make's POV, not a good thing :)
Could probably drop the else part and leave it as default, couldn't it?

This said, I think it'd probably be better to alter paths like this in the Makefile.am instead.